ui/src/common/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const home_page_meta_tag_content =
```

Comment:
It's better to use capital case for constants to be easily understandable [article](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/)